WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -603,7 +605,7 @@ private extension ProductFormViewController {
     func displayError(error: ProductUpdateError?) {
         let title = NSLocalizedString("Cannot update product", comment: "The title of the alert when there is an error updating the product")
 
-        let message = error?.alertMessage
+        let message = error?.localizedDescription
```

Comment:
```suggestion
        let message = error?.errorDescription
```

Reply:
great catch! updated in https://github.com/woocommerce/woocommerce-ios/pull/2781/commits/1b36374546a33d4941e6024156bd2e9b821525e1